src/aggregate_loader.py

```diff
@@ -121,17 +121,17 @@ def run_loader(self):
             ['VARIATION'],
             ['SQTR'],
             ['AGM'],
+            ['DAF'],  # Locks Genes
+            ['ORTHO'],  # Locks Genes
+            ['GeneDiseaseOrtho'],
             ['GFF'],
             ['EXPRESSION'],
             ['ExpressionRibbon'],
             ['ExpressionRibbonOther'],
             ['GENEEEXPRESSIONATLASSITEMAP'],
-            ['DAF'],  # Locks Genes
             ['PHENOTYPE'],  # Locks Genes
-            ['ORTHO'],  # Locks Genes
             ['GAF'],  # Locks Genes
             ['GeoXref'],  # Locks Genes
-            ['GeneDiseaseOrtho'],
```

Comment:
@sierra-moxon did dependencies change? It might be good in the comment to specify the dependency if the order of these matter.

Reply:
I will.